flink-contrib/docker-flink/docker-entrypoint.sh

```diff
@@ -36,9 +39,9 @@ elif [ "$1" == "taskmanager" ]; then
     echo "Starting Task Manager"
     echo "config file: " && grep '^[^\n#]' $FLINK_HOME/conf/flink-conf.yaml
     $FLINK_HOME/bin/taskmanager.sh start
+
+  # prevent script to exit
+  tail -f /dev/null
 else
     $@
```

Comment:
@greghogan Seems like a way to execute an arbitrary command passed inside the Docker container passed as an argument to `docker run <comand>`.